vmm_tests/vmm_tests/tests/tests/x86_64/openhcl_linux_direct.rs

```diff
@@ -155,8 +155,112 @@ fn new_test_vtl2_nvme_device(
+                    vtl2_settings_proto::StorageController {
+                        instance_id: Guid::new_random().to_string(),
+                        protocol: vtl2_settings_proto::storage_controller::StorageProtocol::Scsi
+                            .into(),
+                        luns: device_ids
+                            .iter()
+                            .map(|(nsid, guid)| vtl2_settings_proto::Lun {
+                                location: (*nsid - NSID_OFFSET) + 1, // Add 1 so as to avoid any confusion with booting from LUN 0 (on the implicit SCSI controller created by the above `config.with_vmbus_redirect` call above).
+                                device_id: Guid::new_random().to_string(),
+                                vendor_id: "OpenVMM".to_string(),
+                                product_id: "Disk".to_string(),
+                                product_revision_level: "1.0".to_string(),
+                                serial_number: "0".to_string(),
+                                model_number: "1".to_string(),
+                                physical_devices: Some(vtl2_settings_proto::PhysicalDevices {
+                                    r#type:
+                                        vtl2_settings_proto::physical_devices::BackingType::Single
+                                            .into(),
+                                    device: Some(vtl2_settings_proto::PhysicalDevice {
+                                        device_type:
+                                            vtl2_settings_proto::physical_device::DeviceType::Nvme
+                                                .into(),
+                                        device_path: guid.to_string(),
+                                        sub_device_path: *nsid,
+                                    }),
+                                    devices: Vec::new(),
+                                }),
+                                ..Default::default()
+                            })
+                            .collect(),
+                        io_queue_depth: None,
+                    },
+                )
+            })
+        })
+        .run()
+        .await?;
+
+    for _ in 0..3 {
+        agent.ping().await?;
+
+        // Test that inspect serialization works with the old version.
+        vm.test_inspect_openhcl().await?;
+
+        vm.restart_openhcl(
+            igvm_file.clone(),
+            OpenHclServicingFlags {
+                enable_nvme_keepalive: false,
+                ..Default::default()
+            },
+        )
+        .await?;
+
+        agent.ping().await?;
+
+        // Test that inspect serialization works with the new version.
+        vm.test_inspect_openhcl().await?;
+    }
+
+    agent.power_off().await?;
+    assert_eq!(vm.wait_for_teardown().await?, HaltReason::PowerOff);
+
+    Ok(())
+}
+
+/// Test an OpenHCL Linux direct VM with a SCSI disk assigned to VTL2, an NVMe disk assigned to VTL2, and
```

Comment:
I see the description of this existing test was changed to include an NVMe disk, but the test itself wasn't changed. Was the existing description incorrect?

Reply:
Alright, looks good then